classes/person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parents_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parents_permission = parents_permission
+  end
+
+  def of_age?(age)
+    @age = age
+    @age >= 18
+  end
```

Comment:
- [OPTIONAL] Excellent job creating the Person class, however, it's no necessary to assign `@age = age` everytime you call the method, you can say:

```
def of_age?
    @age >= 18
end
```
And this will have the same result without additional unnecessary logic to keep your code simple.

Reply:
@smunozmo Thank you for the review, I will update the code now.